resources/wikia/modules/iframeWriter.js

```diff
@@ -28,6 +28,11 @@ define('wikia.iframeWriter', [
 		iframe.scrolling = 'no';
 		iframe.width = params.width;
 		iframe.height = params.height;
+
+		if (iframe.classList) {
+			iframe.classList.add('wikia-ad-iframe');
```

Comment:
After giving it a second thought to this I'm wondering if it shouldn't be `wikia-iframe` or if we shouldn't add possibility to pass classes in the `params` object.